drivers/windows/R/paths.R

```diff
@@ -15,11 +14,20 @@ windows_path <- function(name, path_local, path_remote, drive_remote) {
 
   path_remote <- clean_path_remote(path_remote)
   if (!grepl("^\\\\\\\\(.*)$", path_remote)) {
-    stop("path_remote must be a network path, starting with // or \\\\\\\\")
+    cli::cli_abort(c("path_remote must be a network path.",
+                 i = "Network paths should start with // or \\\\\\\\",
+                 i = "Please check {path_remote}"))
   }
 
   if (!file.exists(path_local)) {
-    stop("Local mount point does not exist: ", path_local)
+    cli::cli_abort(c("Local mount point does not exist.",
+                 i = "Please check the mount point: {path_local}."))
+  }
+
+  if (drive_remote == "I:") {
+    cli::cli_abort(c("You cannot use I: on a cluster job.",
+                 i = "I: is reserved for package provisioning.",
```

Comment:
```suggestion
                 i = "I: is reserved for internal use.",
```

More mysterious :shipit:

Reply:
hipercow moves in mysterious ways, its wonders to perform.